modules/openy_features/openy_prgf/modules/openy_prgf_event_latest/openy_prgf_event_latest.info.yml

```diff
@@ -0,0 +1,18 @@
+name: 'OpenY Paragraph Latest Event Posts'
+description: 'OpenY Paragraph Latest Event Posts'
+type: module
+core: 8.x
+package: 'YMCA Maryland'
```

Comment:
provide a version, please

Reply:
@dbuzinov please change package to "OpenY"